src/grt/src/GlobalRouter.tcl

```diff
@@ -282,6 +282,27 @@ proc write_guides { args } {
   grt::write_guides $file_name
 }
 
+sta::define_cmd_args "highlight_route" { net_name }
```

Comment:
the command does not highlight a route or take a net name as an arg anymore so you need to update the names accordingly